ENR-9759 - Release notes (#1219)
corda/corda-docs-portal: content/en/platform/corda/4.10/enterprise/release-notes-enterprise.md

# Corda Enterprise Edition 4.10 release notes

## Corda Enterprise Edition 4.10.1 release notes

Corda Enterprise Edition 4.10.1 is a patch release of Corda Enterprise focused on resolving issues.

### Upgrade recommendation

As a developer or node operator, you should upgrade to the [latest released version of Corda](../enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page, and for the latest upgrade guide, refer to [Upgrading a CorDapp or node](upgrading-index.md).

### Fixed issues

> Corda Enterprise Edition 4.10.1 contains the following fix:

Reviewer: ```suggestion
The following issues were resolved in Corda Enterprise Edition 4.10.1:
```
